src/subscript/config_jobs/RI_WELLMOD

```diff
@@ -0,0 +1,42 @@
+EXECUTABLE  ri_wellmod
```

Comment:
This job configuration warrants an integration test with ERT. Experience has shown that this regularly bites.

Reply:
Agreed - could you point to a good example?

Reply:
https://github.com/equinor/subscript/blob/master/tests/test_csv_stack.py#L245

Reply:
Fantastic - thanks! Stolen with pride.